flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
that's really unfortunate. What's the justification?

Reply:
time how? it should just build. shouldn't it?

Reply:
Would be nice if it did, but I haven't checked it